src/main/java/az/keytd/expensetracker/service/AuthenticationService.java

```diff
@@ -47,4 +54,19 @@ public Response login(LoginRequest request) {
 
         return new Response(200, "ok", token);
     }
+
+    public void verify(String email, String otp) {
+        CommonOtp commonOtp = commonOtpsRepository.findByEmail(email)
+                .orElseThrow(() -> new NotFoundException(email + " not found"));
+
+        if (commonOtp.getOtp() != otp) {
+            int count = commonOtp.getRetryCount() + 1;
+            commonOtp.setRetryCount(count);
+            commonOtpsRepository.save(commonOtp);
+            throw new BadRequestException("your otp code is not true");
+        }
+        commonOtp.setStatus(OtpStatus.OK);
```

Comment:
burda user tap email ile status deyish ele CONFIRMED and save, register-da UNCIRMED elemisen onda gore deyishemk lazimdiki bilek bu mushteri tesdiqledin